stl/src/special_math.cpp

```diff
@@ -17,8 +17,17 @@
 #define BOOST_MATH_DOMAIN_ERROR_POLICY   errno_on_error
 #define BOOST_MATH_OVERFLOW_ERROR_POLICY ignore_error
 
-// Using headers from Boost.Math 1.66.0
-#include <boost/math/special_functions.hpp>
+// Using headers from Boost.Math
+#include <boost/math/special_functions/bessel.hpp>
+#include <boost/math/special_functions/beta.hpp>
+#include <boost/math/special_functions/ellint_1.hpp>
+#include <boost/math/special_functions/ellint_2.hpp>
+#include <boost/math/special_functions/ellint_3.hpp>
+#include <boost/math/special_functions/expint.hpp>
+#include <boost/math/special_functions/hermite.hpp>
+#include <boost/math/special_functions/laguerre.hpp>
+#include <boost/math/special_functions/legendre.hpp>
+#include <boost/math/special_functions/spherical_harmonic.hpp>

```

Comment:
You're missing `#include <boost/math/special_functions/zeta.hpp>` for

https://github.com/microsoft/STL/blob/8dc0385763f947354b23dbb217d3878d412026ec/stl/src/special_math.cpp#L394

as documented by: https://www.boost.org/doc/libs/1_72_0/libs/math/doc/html/math_toolkit/zetas/zeta.html